WooCommerce/WooCommerceTests/ViewRelated/Products/ProductDetailsFactoryTests.swift

```diff
@@ -85,34 +42,14 @@ final class ProductDetailsFactoryTests: XCTestCase {
 
     // MARK: Grouped product type
 
-    func test_factory_creates_product_form_for_grouped_product_when_products_release_3_is_on() {
-        // Arrange
-        let mockStoresManager = MockProductsAppSettingsStoresManager(isProductsFeatureSwitchEnabled: true, sessionManager: SessionManager.testingInstance)
-        let product = Product.fake().copy(productTypeKey: ProductType.grouped.rawValue)
-        let expectation = self.expectation(description: "Wait for loading Products feature switch from app settings")
-
-        // Action
-        ProductDetailsFactory.productDetails(product: product,
-                                             presentationStyle: .navigationStack,
-                                             stores: mockStoresManager,
-                                             forceReadOnly: false) { viewController in
-                                                // Assert
-                                                XCTAssertTrue(viewController is ProductFormViewController<ProductFormViewModel>)
-                                                expectation.fulfill()
-        }
-        waitForExpectations(timeout: Constants.expectationTimeout, handler: nil)
-    }
-
-    func test_factory_creates_product_form_for_grouped_product_when_products_release_3_is_off() {
+    func test_factory_creates_product_form_for_grouped_product() {
```

Comment:
The `expectation` below (`Wait for loading Products feature switch from app settings`) should be updated as well (3)